crip/shared.py

```diff
@@ -55,3 +57,33 @@ def gaussianSmooth(projection, sigma, ksize=None):
     if isinstance(sigma, int):
         sigma = (sigma, sigma)
     return cv2.GaussianBlur(projection.astype(np.float32), ksize, sigmaX=sigma[0], sigmaY=sigma[1])
+
+
+def verticalFlip(img):
+    '''
+        Vertical flip one image.
+    '''
+    cripAssert(len(img.shape) == 2, 'verticalFlip supports only 2D image.')
+
+    return np.flipud(img)
+
+
+def horizontalFlip(img):
+    '''
+        Horizontal flip one image.
+    '''
+    cripAssert(len(img.shape) == 2, 'horizontalFlip supports only 2D image.')
+
+    return np.fliplr(img)
+
+
+def stackImages(imgList, dtype='keep'):
```

Comment:
Unlike [this](https://github.com/z0gSh1u/crip/blob/e351387d1f7d0f6cad5738f871e0e434f130fcd8/crip/io.py#L28):

```
def combineRawImageUnderDirectory(folder, h, w, dtype=np.float32, nSlice=1, offset=0, sort='nat', reverse=False):
```

We'd like every method does exactly one job and dont "flatter" users so that they can customize their own pipeline at the biggest extent.